src/main.cpp

```diff
@@ -3125,7 +3126,7 @@ bool CheckBlock(const CBlock& block, CValidationState& state, bool fCheckPOW, bo
     {
         nSigOps += GetLegacySigOpCount(tx);
     }
-    if (nSigOps > MAX_BLOCK_SIGOPS)
+    if (nSigOps > Policy::blockSigOpAcceptLimit(blockSize))
```

Comment:
This is how Bitcoin counts non-p2sh addresses, yes.

Why do you think BU changed that, and if you feel that that is the case, that is something to bring up with BU.